bioptim/limits/constraints.py

```diff
@@ -94,6 +94,15 @@ def print(self):
         # TODO: Print all elements in the console
         raise NotImplementedError("Printing of ConstraintList is not ready yet")
 
+    @staticmethod
+    def get_nlp_constraints(nlp):
+        list_constraints = [[{"Constraints": []} for _ in range(NLP.ns + 1)] for NLP in nlp]
```

Comment:
is nlp here refers to all the nlp of the ocp? If this is the case, we used the prefix `all_` so far, like this you can use non capitalized letters for the current nlp

Reply:
Because like this, it feels like NLP is supposed to be a module haha

Reply:
Yes, nlp refers to all the nlp of the ocp. I directly sent the ocp instead of the nlp as parameter of the function. The problem is solved.

Reply:
Done